### PR TITLE
Features/ccs 748 test for empty filtering

### PR DESCRIPTION
**Pull request by dmelosantos**
Adding tests for filtering empty credentials

### CODE REVIEW

File: __test__/creds/VerifiableCredential.test.js [hunk 1]
```diff
@@ -844,7 +844,7 @@ describe('Unit tests for Verifiable Credentials', () => {
       await credential.updateAnchor();
 
       const filteredCredential = credential.filter([]);
-      return Object.keys(filteredCredential.claim).length === 0;
+      return Object.keys(filteredCredential.claim).length === 0 && filteredCredential.verify() === 0;
```
[the remark this answers is not in the document]
Comment (dmelosantos) [reply __test__/creds/VerifiableCredential.test.js]: Changed
